pkg/clients/eks/eks.go

```diff
@@ -1,11 +1,11 @@
 /*
-Copyright 2019 The Crossplane Authors.
+Copyright 2020 The Crossplane Authors.
```

Comment:
@muvaf yes, what happened here is that the old file was renamed, then this one got created so it appears as a diff rather than a new file. I am happy to change it back to `2019` if you think necessary.